### PR TITLE
Issue 945: Sqlite with multiple primary keys

### LINKED ISSUE #945: Sqlite: composite primary key ignored on synchronization
When using two columns as primary key in sqlite like this:
```typescript
import { Entity, PrimaryColumn, Column } from 'typeorm';

@Entity()
export class ExampleEntity {
  @PrimaryColumn()
  id1: string;

  @PrimaryColumn()
  id2: string;

  @Column()
  otherColumn: boolean;
}
```
When making a change a creation sql like this is generated:
```sql
CREATE TABLE "temporary_example_entity"
("id1" varchar NOT NULL, "id2" varchar NOT NULL, "otherColumn" boolean, PRIMARY KEY(id1))
```
But is should be
```sql
CREATE TABLE "temporary_example_entity"
("id1" varchar NOT NULL, "id2" varchar NOT NULL, "otherColumn" boolean, PRIMARY KEY(id1,id2))
```

This issue only exists when syncing. The initial creation is correct.

### PR DESCRIPTION
**Pull request by daniel-lang**
fixes #945

### CODE REVIEW

**Comment by daniel-lang** [comment]
According to the [sqlite docs](https://sqlite.org/pragma.html#pragma_table_info) the column "pk" in `pragma table_info()` is 0 when the column isn't a primary key and otherwise the index

**Comment by pleerock** [comment]
Looks like test is failing

**Comment by daniel-lang** [comment]
It is failing for postgres. I currently checking why

**Comment by daniel-lang** [comment]
For the postgres drivers the columns aren't marked with `isPrimary`. But the `primaryKeys` array includes the correct keys. If this is by design, you can merge this PR

File: test/github-issues/945/issue-945.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        dropSchema: true
+    }));
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+
+    it("schema should include two primary keys", () => Promise.all(connections.map(async connection => {
+        const tableSchema = await connection.createQueryRunner().getTable("test_entity");
```
Comment (pleerock) [inline test/github-issues/945/issue-945.ts]: when you create a query runner, you must always release it

**Comment by daniel-lang** [comment]
Thank you for pointing that out. Is the postgres behaviour indended?

**Comment by pleerock** [comment]
I think no. That part is quite old and needs refactoring. Most probably its a bug

**Comment by daniel-lang** [comment]
Okay, so this request is good to go or should I investigate the problem?

**Pull request merged**